<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Skylark.local" name="hub.test.RegressionSuite.AdminCartAndPayment" tests="3" failures="1" timestamp="23 Jul 2014 09:36:50 GMT" time="216.913" errors="0">
  <testcase name="Configurable_Account_Password" time="151.888" classname="hub.test.RegressionSuite.AdminCartAndPayment"/>
  <testcase name="Account_Password_OFF_Payment_Details_validation" time="64.945" classname="hub.test.RegressionSuite.AdminCartAndPayment">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at hub.utilities.EndToEnd.UserHubEndToEndUtil_SmokeTest.SuccessfulLogin(UserHubEndToEndUtil_SmokeTest.java:263)
	at hub.test.RegressionSuite.AdminCartAndPayment.Account_Password_OFF_Payment_Details_validation(AdminCartAndPayment.java:56)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- Account_Password_OFF_Payment_Details_validation -->
  <testcase name="Validation_AccountPassword_InitialValue" time="0.080" classname="hub.test.RegressionSuite.AdminCartAndPayment"/>
</testsuite> <!-- hub.test.RegressionSuite.AdminCartAndPayment -->
